src/adapter/markdown_adapter.erl

```diff
@@ -31,5 +31,12 @@
 
 -spec adapt(binary()) -> iodata().
 adapt(Line) ->
-    Line1 = emarkdown:to_html(Line),
-    html_adapter:adapt(Line1).
+    %% Prepare links
+    L0 = binary:replace(Line,
+                        [<<"`">>, <<"[">>, <<"]">>, <<"(">>, <<")">>],
+                        <<" ">>,
+                        [global]),
+    %% Delete links in line
+    L1 = re:replace(L0, "(http\S*?)(\.*?\s)", " ", [global, {return, binary}]),
+    %% Delete other symbols
+    binary:replace(L1, [<<"`">>, <<"<">>, <<">">>, <<"*">>, <<"#">>], <<>>, [global]).
```

Comment:
What about `~`. Should we account for that one as well?

Reply:
well, I suppose in future we will add more changes here because I don't see trim of `~` and some other chars in `sheldon`.

Reply:
Current PR is focused on fixing issue of Windows compile + CT passing without code changes in test cases - I suppose this is a good start.

Reply:
BTW:
```erlang
1> sheldon:check("~").
#{bazinga =>
      <<"The only way you'd be able to make a contribution to science is if they resume sending chimps into space.">>,
  misspelled_words =>
      [#{candidates =>
             ["2","A","a","B","b","C","c","D","d","E","e","F","f","G",
              "g","H","h","I","i","J","j","K","k","L","l",
              [...]|...],
         line_number => 1,word => "~"}]}
```